IDE/MSVS-2019-AZSPHERE/client/client.c

```diff
@@ -0,0 +1,138 @@
+﻿/* client.c
```

Comment:
Is this folder name with MSVC and 2019 ideal? Seems very specific. Would prefer generic IDE/AzSphere` or something.

Reply:
Yes this is a good idea. The Azure team changed completely how the build tools work between MSVS 2017 and MSVS 2019. 2019 can't build the  IDE/VS-AZURE-SPHERE/. The 2017 Azure Sphere plugin to MSVS doesn't support the new cmake builds We're adding in MSVS-2019-AZSPHERE.

Given this massive change from 2017 to 2019 I would hesitate to place any version but 2019 on the example because I don't know that it will still work by the time we upgrade our CI system to use 2021 or any future MSVS versions.